src/types/types.ts

```diff
@@ -426,6 +426,14 @@ export interface EdgeTokenInfo {
   multiplier: string
 }
 
+/**
+ * Search the blockchain for a token.
+ */
+export interface EdgeGetTokenDetails {
```

Comment:
Also, shouldn't this be named `EdgeGetTokenDetailsParams` or `EdgeGetTokenDetailsFilter`?